src/views/Workspace/Cards/DonateCard.jsx

```diff
@@ -0,0 +1,414 @@
+
+    return viewName === inOut;
+};
+
+// step:  "Doneer"
+class DonateCard extends Component {
+    state = {
+        dialogOpen: false,
+        bank: 'ABN-AMRO',
+        sum: 25,
+        comment: '',
+        isAnonymousDonation: true,
+        isMoneyToAnotherActie: true,
+        view: 'beide',
+    };
+
+    toggleDialog = () => this.setState(({ dialogOpen }) => ({ dialogOpen: !dialogOpen }));
+
+    changeText = ({ target: { name, value } }) => this.setState({ [name]: value });
+    changeSum = ({ target: { name, value } }) => this.setState({ [name]: value.replace(/(-|\+)/, '') });
+    handleCheck = (name) => () => this.setState({ [name]: !this.state[name] });
+
+    changeView = (view) => () => this.setState({ view });
+
+    submit = () => {
+
+    };
+
+    render() {
+        const { classes } = this.props;
+        const { bank, comment, isAnonymousDonation, isMoneyToAnotherActie, sum, dialogOpen, view } = this.state;
+
+        return (
+          <div>
+              <WorkspaceCard
+                title={"Doneer"}
+                image={require("assets/img/backgrounds/workspace-welcome-bg.jpg")}
+                content={""}
+                expandContent={
+                    <div>
+                        <div className={classes.content}>
+                            <div className={classes.donation}>
+                                <div className={classes.totalSum}>€{mocks.totalSum}</div>
+                                <p className={classes.totalSubtitle}>Opgehaald bedrag</p>
+                                <Button onClick={this.toggleDialog} className={classes.donateButton} variant="outlined">
+                                    <DonateIcon className={classes.donateIcon} />
+                                    <div className={classes.donateText}>Doneer</div>
+                                </Button>
+                            </div>
+
+                            <div className={classes.description}>
+                                <Typography component="p" className={classes.title}>Aangesloten producten*:</Typography>
+
+                                <ul>
+                                    <li className={classes.listItem}>Flyers</li>
+                                    <li className={classes.listItem}>Posters</li>
+                                    <li className={classes.listItem}>Borden</li>
+                                </ul>
+
+                                <Typography component="p" className={classes.text}>Vrijwilligers kunnen het geld ultgeven aan bovenstaande producten. Indien het geld niet wordt uitgegeven kan het gestort worden aan de aaliantle Nederland rookvrij. Deze keuze maak je wanneer je doneert</Typography>
+                            </div>
+                        </div>
+
+                        <div className={classes.donationsTop}>
+                            <div className={classes.donationsTitle}>Transacties in uit beide</div>
```

Comment:
Transacties 

remove "in uit beide" from title

Reply:
Done!